packages/mikmatch/mikmatch.1.0.9/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "ygrek@autistici.org"
+homepage: "https://mjambon.github.io/mjambon2016/micmatch.html"
+doc: ["https://mjambon.github.io/mjambon2016/mikmatch-manual.html"]
+bug-reports: "https://github.com/mjambon/mikmatch/issues"
+dev-repo: "git://github.com/mjambon/mikmatch.git"
+authors: [ "Martin Jambon" ]
+build: [
+  [make "str"]
+  [make] {"%{pcre:installed}%"}
+]
+install: [
+  [make "install-str"]
+  [make "install"] {"%{pcre:installed}%"}
+]
+remove: [
+  [make "uninstall-str"]
+  [make "uninstall"] {"%{pcre:installed}%"}
+]
+depends: [
+  "ocaml" {>= "4.07"}
+  "ocamlfind" {build}
+  "camlp4" {build}
```

Comment:
Looking at https://github.com/mjambon/mikmatch/blob/23f78d273dbcc4c1b0a38beddffb0e0f3991d8a7/pcre/META it seems that camlp4 is used at runtime as well.

Reply:
Also know that camlp4 is not available for OCaml 4.09 and above and is deprecated.

Reply:
https://github.com/ocaml/camlp4/pull/153

Reply:
oh i see. Thanks for the info